### DISCUSSION TITLE
QR not generated on Google RUN Console Cloud

### DISCUSSION THREAD

**Discussion by beshoo** [devlikeapro/waha#690] [General]
I have this Google instance, "CPU only allocated during requests"

![image](https://github.com/user-attachments/assets/4312a46b-a7e5-4ffb-84f5-9bb6937577e2)

Sometimes I have this error:
![image](https://github.com/user-attachments/assets/3b9129e6-0b8c-4388-9326-02469c1fb680)

I'm not sure why the QR code is not available. Is it because the instance is "CPU only allocated during requests"? This means the CPU only runs when you send a request, but the timeout is 300 seconds.

Please advise!

**Comment by devlikepro** [comment]
Hi! 
Yeah, looks like it's the issue - you can send "GET /api/server/version" every 10 seconds to keep instance alive or consider getting any VPS and run it there

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> I'm not sure why the QR code is not available

About that - try setting up proxy https://waha.devlike.pro/docs/how-to/proxy/ or increase CPU limits for that instance.

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
